Examples/HelloWorld/CMakeLists.txt

```diff
@@ -6,3 +6,45 @@ target_link_libraries ( HelloWorld ${SimpleITK_LIBRARIES} )
 if ( WRAP_CSHARP )
   csharp_add_executable( HelloWorldCS SimpleITKCSharpManaged.dll ${CMAKE_CURRENT_SOURCE_DIR}/HelloWorld.cs )
```

Comment:
I'm not using the sitk_add_charp_test.  It's actually using sitk_add_test.  I wanted the Csharp to compile even without testing.